dockerfiles/verify_packages/installer/test_alpine_no_ext_json.sh

```diff
@@ -10,10 +10,10 @@ apk add php7 curl libexecinfo
 assert_no_ddtrace
 
 # Install using the php installer
-new_version="0.65.1"
+new_version="0.68.0"
```

Comment:
I tricked the system. This comes from https://github.com/labbati/test-actions/releases to run tests, then it will be changed to dd-trace-php before merging